### PR TITLE
Show more information in terminator examples

### PR DESCRIPTION
**Pull request by HideakiImamura**
## Motivation

This is a follow-up of #190. Showing more information in terminator examples and use `if __name__ == "__main__"`.

## Description of the changes

- Show more information.
- Use `if __name__ == "__main__"`.
- Fix CI conditions.

### CODE REVIEW

**Comment by HideakiImamura** [comment]
@gen740 Could you review this PR?

**Review by gen740 — APPROVED** [review]
LGTM! Thank you very much for the follow-up PR of my PR!

File: terminator/terminator_simple.py [hunk 1]
```diff
@@ -37,5 +37,9 @@ def objective(trial):
     return scores.mean()
 
 
-study = optuna.create_study(direction="maximize")
-study.optimize(objective, callbacks=[TerminatorCallback()])
+if __name__ == "__main__":
+    study = optuna.create_study(direction="maximize")
+    study.optimize(objective, callbacks=[TerminatorCallback()])
```
Comment (gen740) [inline terminator/terminator_simple.py:42]:
```suggestion
    study.optimize(objective, n_trials=50, callbacks=[TerminatorCallback()])
```
I've realized that specifying `n_trials` is better in cases where the trials are not terminated.

**Review by gen740 — COMMENTED** [review]
Thank you very much for the follow-up PR of my PR!
Almost LGTM! But I have a minor comment.

**Pull request merged**